internal/config/config.go

```diff
@@ -21,6 +21,7 @@ import (
 const (
 	DefaultGraphQLEndpoint     = "https://graphql.buildkite.com/v1"
 	OrganizationsSlugConfigKey = "organizations"
+	OpenAITokenKey             = "openai_token"
```

Comment:
We don't need to access things through these externally (and its not used internally anyway so we can remove it